app/javascript/src/components/Dashboard/Contacts/ContactsTable.jsx

```diff
@@ -0,0 +1,98 @@
+import React from "react";
+import { Checkbox, Avatar, Button, Tooltip } from "neetoui";
+
+const ContactsTable = ({
+  selectedContactIds,
+  setSelectedContactIds,
+  contacts,
+}) => {
+  return (
+    <div className="w-full px-4">
+      <table className="nui-table nui-table--checkbox nui-table--actions">
+        <thead>
+          <tr>
+            <th>
+              <Checkbox
+                checked={
+                  selectedContactIds.length ===
+                  contacts.map(contact => contact.id).length
+                }
+                onClick={() => {
+                  const noteIds = contacts.map(contact => contact.id);
+                  if (selectedContactIds.length === noteIds.length) {
+                    setSelectedContactIds([]);
+                  } else {
+                    setSelectedContactIds(noteIds);
+                  }
+                }}
+              />
+            </th>
+            <th className="text-left text-gray-400">name</th>
+            <th className="text-left text-gray-400">email</th>
+            <th className="text-center text-gray-400">department</th>
+            <th className="text-center text-gray-400">contact number</th>
+            <th className="text-center text-gray-400">add to basecamp</th>
```

Comment:
Please remove `text-gray-400`.

Reply:
The `text-gray-400` matches with the heading text color in the design. Do you want me to remove it anyway?

Reply:
Yes.